apps/docs/content/guides/auth/enterprise-sso/auth-sso-saml.mdx

```diff
@@ -421,3 +421,47 @@ If you really need access to the key, please [open a support ticket](https://sup
 Yes, Supabase supports multi-tenant Single Sign-On (SSO) using SAML 2.0. While the dashboard displays only one SAML field, you can set up multiple SAML connections using the Supabase CLI.
 Each connection is assigned a unique `sso_provider_id`, which is included in the user's JWT and can be used in Row Level Security (RLS) policies. You can configure custom attribute mappings for each connection to include tenant-specific information, such as roles.
 This setup allows you to implement multi-tenant SSO for multiple clients or organizations within a single application. For example, if you have an app with multiple clients using different Azure Active Directories, you can create separate SAML connections for each and use the `sso_provider_id` to manage access and apply appropriate security policies.
+
+### Is multi-subdomain SSO with SAML supported?
+
+Yes, also referred to as [cross-origin authentication within the same site](https://web.dev/articles/same-site-same-origin). To redirect to a URL other than the [Site URL](https://supabase.com/docs/guides/auth/redirect-urls), following the SAML response from the IdP, the `redirectTo` option can be added to [`signInWithSSO`](https://supabase.com/docs/reference/javascript/auth-signinwithsso).
```

Comment:
Might be worth mentioning that the redirect URL should be in the allowed redirect users. At the same time, since SSO users are expected a little more advanced I think it's fine to do without and maybe add the caveat if we see any tickets with such an issue

Reply:
oh good idea, yeah. Putting it as a link on Site URL probably wasn't the best place. Something like,

> ...following the SAML response from the IdP, the `redirectTo` option can be added to `signInWithSSO`. **Make sure to add the redirect URL to the approved list in your project dashboard.**